app/assets/stylesheets/sign_in.scss

```diff
@@ -0,0 +1,10 @@
+@import "colors";
+
+.sign-up {
+  background-image: url("wallpaper.jpg");
+  border-radius: 20px;
+  padding-bottom: 100px;
```

Comment:
Properties should be ordered background-image, border-radius, box-shadow, padding-bottom, padding-top, text-align